### PR TITLE
Updating housing methane emissions in manure handlers

### PR DESCRIPTION
**Pull request by ltp23**
This PR aims to update the calculation of housing methane emissions in the manure handlers. 

The changes happen in the two files `manure_handler_classes.py` and `gas_emission.py`.

This is part of the bigger PR [#563](https://github.com/RuminantFarmSystems/MASM/pull/563), which is now in draft state.

### CODE REVIEW

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py [hunk 1]
```diff
@@ -92,26 +92,43 @@ def _calc_ambient_temp(cls, hours: float, temperature_min: float, temperature_ma
...
+        """
+        Calculate housing methane emissions from manure handlers.
 
-        Args:
-            num_animals: Number of animals in the pen.
-            barn_area: Area of the barn based on housing type, m^2.
-            hours: hours of the day from 1 to 24.
-            temperature_min: Minimum barn temperature, C.
-            temperature_max: Maximum barn temperature, C.
+        The equation used to calculate housing methane emissions is:
```
Comment (ew3361zh) [inline RUFAS/routines/manure/gas_emissions/gas_emissions.py:102]: I believe equations/formulas should go in the Notes section of the docstrings as opposed to the initial short summary section:

https://numpydoc.readthedocs.io/en/latest/format.html#notes

**Review by ew3361zh — APPROVED** [review]
Really nice work, Loi, thank you for breaking that larger PR up into these smaller parts. It's very helpful and appreciated!

**Review by varrma — APPROVED** [review]

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py (hunk 1 above)
Comment (ltp23) [reply RUFAS/routines/manure/gas_emissions/gas_emissions.py:102]: Thanks Niko. I'll add a notes section for formulas from now on. That's a good link.

**Comment by ltp23** [comment]
Thanks everyone!

**Pull request merged**